lava-master/Dockerfile

```diff
@@ -36,6 +39,9 @@ COPY env/ /etc/lava-server/dispatcher.d/
 
 COPY apache2/ /etc/apache2/
 
+# Thoses two line fixes 'postgresql ERROR:  invalid locale name: "en_US.UTF-8"' when restoring a backup
```

Comment:
which two lines?  I only see one line added?

Reply:
I compressed thoses two lines in one, I update the comment